flipperdevices/Flipper-Android-App
# Infrared backend integration v2

**Background**

This PR updates backend integration and fixes buttons display

**Changes**

Here what change in this PR..
- Update backend models
- Update buttons display 
- 
**Test plan**

- Open infrared finding feature
- Try to use it
- Buttons should be update now and the feture overall should works better
